src/components/program/ProgramFAQ.jsx

```diff
@@ -0,0 +1,36 @@
+import { useContext } from 'react';
+import { Collapsible } from '@edx/paragon';
+import { faChevronUp, faChevronDown } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { ProgramContext } from './ProgramContextProvider';
+import './styles/index.scss';
+
+function ProgramFAQ() {
+  const { program: { faq } } = useContext(ProgramContext);
+  if (faq && faq.length > 0) {
+    return (
+      <div className="col-lg-8 p-2">
+        <h2 style={{ color: '#001b1e' }}>FAQs</h2>
```

Comment:
nit: this color property can be defined in a constant somewhere and then called here + inside `ProgramEndorsements` component.